extensions/sql/CMakeLists.txt

```diff
@@ -89,11 +84,12 @@ find_package(Patch)
 set(PC "${Patch_EXECUTABLE}" -p1 -i "${CMAKE_CURRENT_SOURCE_DIR}/patch/soci.patch")
 
 # Define byproducts
-if(NOT APPLE AND CMAKE_SIZEOF_VOID_P EQUAL 8)
-	include(GNUInstallDirs)
-	set(LIBDIR "${CMAKE_INSTALL_LIBDIR}")
-else()
+# This should be based on GNUInstallDirs, but it's done wrong in Soci:
+# https://github.com/SOCI/soci/blob/release/4.0/CMakeLists.txt#L140
+if(APPLE OR CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
Shouldn't this be always "lib" for WIN32 as well?

Reply:
This is now consistent with what the lib does internally (see the link in the comment above), so I assume this to be correct for Win as well.